src/model/nonce.ts

```diff
@@ -32,3 +33,18 @@ export const create = (nonce: Nonce) => (
     singleStringReply,
     falsyResponseToErrorAsync(new Error("Error saving the key"))
   );
+
+type GetDeleteNonceT = (
+  nonce: Nonce
+) => (redis_client: redis.RedisClientType) => TE.TaskEither<Error, true>;
+export const getDeleteNonce: GetDeleteNonceT = nonce => redis_client =>
```

Comment:
Should we use the same name pattern for the model actions and the method signature can be embedded into the declaration instead to creare a custom type.
```suggestion
export const invalidate = (nonce: Nonce) => (
  redisClient: redis.RedisClientType
): TE.TaskEither<Error, true> =>
```

Reply:
done in https://github.com/pagopa/io-functions-fast-login/pull/27/commits/afa278b7e7d61fa946dc35679329250566193fdd